Assets/Scripts/Obstacles/Rails.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Input;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Serialization;
+using UnityEngine.Splines;
+
+namespace Obstacles
+{
+    [RequireComponent(typeof(Collider))]
+    public sealed class Rails : MonoBehaviour
+    {
+        [SerializeField] private CameraPositioner _cameraPositioner;
+        [SerializeField] private SplineAnimate _splineAnimate;
+        [SerializeField] private Transform _endPoint;
+        [SerializeField] private InputToggle _input;
+
+        [SerializeField] private GameObject _player;
+        [SerializeField] private GameObject _ball;
+        
+        private Rigidbody _rigidbody;
+
+        private void Start()
+        {
+            _rigidbody = _player.GetComponent<Rigidbody>();
+        }
+        
+        private IEnumerator ActivateRails()
+        {
+            ToggleVisuals();
+            yield return new WaitForSeconds(0.05f);
```

Comment:
Deze random `0.05f` kan beter een private const zijn.

Reply:
is aleen voor dat de animatie niet tript niet echt value die belangrijk is